src/main/java/codeu/model/data/User.java

```diff
@@ -24,44 +24,86 @@
   private final String passwordHash;
   private final Instant creation;
   private final Boolean isAdmin;
+  private final String bio;
+  private final String instagramHandle;
+  private final String snapchatHandle;
 
   /**
    * Constructs a new User.
    *
-   * @param id the ID of this User
-   * @param name the username of this User
-   * @param passwordHash the password hash of this User
-   * @param creation the creation time of this User
+   * @param id              the ID of this User
+   * @param name            the username of this User
+   * @param passwordHash    the password hash of this User
+   * @param creation        the creation time of this User
    */
   public User(UUID id, String name, String passwordHash, Instant creation, Boolean isAdmin) {
-    this.id = id;
-    this.name = name;
-    this.passwordHash = passwordHash;
-    this.creation = creation;
-    this.isAdmin = isAdmin;
+    this(id, name, passwordHash, creation, isAdmin, null, null, null);
```

Comment:
@blessinga98 This is how constructor chaining/overloading looks like. Notice that we deleted all the code from @ivannpe 's previous constructor, and called the new constructor, which accepts more parameters. The constructors with less params will almost always call the one with the most params